services/portal/home/homeTasks.js

```diff
@@ -52,30 +46,20 @@ module.exports = {
    * /!\ remember to logout after logging in or following tests will fail!
    */
   async login(username) {
-    this.goToHomepage();
-    
-    // if `systemUse.showOnlyOnLogin` is false, the system use message is _before_ login
-    await this.systemUseMsg();
-
     I.setCookie({ name: 'dev_login', value: username });
     portal.clickProp(homeProps.googleLoginButton);
-    
-     // if `systemUse.showOnlyOnLogin` is true, the system use message is _after_ login
```

Comment:
could you add these comments back in `homeSequences` so we know why we're doing it twice?